datahub-web-react/src/app/onboarding/config/IngestionOnboardingConfig.tsx

```diff
@@ -13,9 +13,9 @@ export const IngestionOnboardingConfig: OnboardingStep[] = [
         content: (
             <Typography.Paragraph>
                 <p>
-                    Click here to configure new Integrations from DataHub to your <strong>Data Platforms</strong>,
-                    including Transactional Databases like <strong>MySQL</strong>, Data Warehouses like{' '}
-                    <strong>Snowflake</strong>, Dashboarding tools like <strong>Looker</strong>, and many more!
+                    Configure new Integrations from DataHub to your <strong>Data Platforms</strong>, including
+                    Transactional Databases like <strong>MySQL</strong>, Data Warehouses such as{' '}
+                    <strong>Snowflake</strong>, Dashboarding tools like <strong>Looker</strong>, and more!
```

Comment:
Wording should be consistent for dashboarding, we should say "Dashboarding tools such as Looker"

Reply:
I disagree. Typically by changing it sounds less repetitive, more natural